tests/README.md

```diff
@@ -69,5 +107,6 @@ When adding features:
 1. Add tests to appropriate file
 2. Follow naming convention: `test_*`
 3. Use existing fixtures
-4. Ensure tests pass locally
+4. Handle both amdsmi package and ctypes backends
```

Comment:
So this is a place where the implementation raises to requirements question - is it necessary to keep supporting the ctypes backend?

If we do require both, it will be hard to maintain parity - there are lots of different conditional codepaths, and we might end up with situations where customers are getting different answers based on whether they have the amdsmi package.

Do we need to maintain ctypes backend support even at the risk of drift here? If pyrsmi has some freedom to choose here, it might be a lot easier, now and long-term, to pick amdsmi as a single dependency.